2-environments/README.md

```diff
@@ -42,7 +42,7 @@ The purpose of this step is to set updevelopment,non-production, and production
     _TF_SA_EMAIL
     _STATE_BUCKET_NAME
     ```
-1. Copy terraform wrapper script `cp ../terraform-example-foundation/build/tf-wrapper.sh . `1. Copy terraform wrapper script `cp ../terraform-example-foundation/build/tf-wrapper.sh . ` to the root of your new repository  (modify accordingly based on your current directory). to the root of your new repository (modify accordingly based on your current directory).
+1. Copy terraform wrapper script `cp ../terraform-example-foundation/build/tf-wrapper.sh . ` to the root of your new repository  (modify accordingly based on your current directory). to the root of your new repository (modify accordingly based on your current directory).
```

Comment:
The phrase `to the root of your new repository (modify accordingly based on your current directory).` is repeated twice.